app/src/main/res/layout-sw600dp-land/profile_chooser_profile_view.xml

```diff
@@ -59,38 +59,29 @@
 
         <TextView
           android:id="@+id/profile_name_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
+          style="@style/Heading2"
           android:ellipsize="end"
-          android:fontFamily="sans-serif-medium"
           android:gravity="center"
           android:maxLines="2"
           android:singleLine="false"
           android:text="@{viewModel.profile.name}"
-          android:textColor="@color/white"
-          android:textSize="20sp" />
+          android:textColor="@color/white"/>
 
         <TextView
           android:id="@+id/profile_last_visited"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:fontFamily="sans-serif-light"
```

Comment:
Similarly, we used light before so ``Subtitle1`` doesn't seem like the correct fit.

@rt4914 looking at https://github.com/oppia/oppia-android/blob/a44791096a3e797653c3d463158db6e7662ab5cb/app/src/main/res/values/styles.xml we don't seem to have any styles with ``sans-serif-light``. Are the current layouts wrong, or are we missing some styles?

Reply:
Done

Reply:
Thanks. Holding off on resolving this until @rt4914 can weigh in since this particular situation probably should have a new style given you found a bunch of places where the same properties are being used.